ide/git/src/org/netbeans/modules/git/ui/tag/ManageTagsAction.java

```diff
@@ -47,24 +47,34 @@
 public class ManageTagsAction extends SingleRepositoryAction {
 
     private static final Logger LOG = Logger.getLogger(ManageTagsAction.class.getName());
+    private static final String ICON_RESOURCE = "org/netbeans/modules/git/resources/icons/tags.png"; //NOI18N
+
+    public ManageTagsAction() {
+        super(ICON_RESOURCE);
+    }
+
+    @Override
+    protected String iconResource() {
+        return ICON_RESOURCE;
+    }
 
     @Override
-    protected void performAction (File repository, File[] roots, VCSContext context) {
+    protected void performAction(File repository, File[] roots, VCSContext context) {
```

Comment:
The formatting changes here and below should be reverted.

Reply:
I reverted the formatting changes, I hope that the new commit is ok...